instaclustr/resource_kafka_schema_registry_user.go

```diff
@@ -0,0 +1,76 @@
+package instaclustr
+
+import (
+"encoding/json"
+"fmt"
+"log"
+
+"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceKafkaSchemaRegistry() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceKafkaSchemaRegistryUserRead,
```

Comment:
Resources should have an implementation for Create as well.

Reply:
Will implement that. 👍

Reply:
Done.